Server/AutomationActions/Gestures/SetPickerWheelValue.m

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#import "Application.h"
+#import "CBXConstants.h"
+#import "CBX-XCTest-Umbrella.h"
+#import "CoordinateQueryConfiguration.h"
+#import "GestureConfiguration.h"
+#import "InvalidArgumentException.h"
+#import "SetPickerWheelValue.h"
+#import "Testmanagerd.h"
+#import "ThreadUtils.h"
+#import "Query.h"
+#import "QueryConfiguration.h"
+#import <UIKit/UIKit.h>
+#import "XCTest+CBXAdditions.h"
+
+@implementation SetPickerWheelValue
+
++ (NSString *)name { return @"set_picker_wheel_value"; }
+
++ (NSArray <NSString *> *)requiredKeys { return @[ CBX_PICKER_INDEX_KEY, CBX_PICKER_WHEEL_INDEX_KEY, CBX_PICKER_WHEEL_VALUE_KEY ]; }
+
++ (NSArray <NSString *> *)optionalKeys {
+    return @[];
+}
+
+- (void)validate {
+    if ([self.query.queryConfiguration isCoordinateQuery]) {
+        return;
+    }
+}
+
++ (Gesture *)executeWithGestureConfiguration:(GestureConfiguration *)gestureConfig query:(Query *)query completion:(CompletionBlock)completion {
```

Comment:
this method never returns Gesture object? Is it ok?

Reply:
This is how Gesture class implemented. I am using Gesture interface to create this class. For now I didn't wanted to create separate class for that.